examples/dodge-the-creeps/rust/src/hud.rs

```diff
@@ -26,12 +26,17 @@ impl Hud {
     pub fn show_game_over(&self) {
         self.show_message("Game Over".into());
 
+        let mut timer = self.base.get_tree().unwrap().create_timer(2.0).unwrap();
+        timer.connect("timeout".into(), self.base.callable("_show_start_button"));
+    }
+
+    #[func]
+    fn _show_start_button(&mut self) {
         let mut message_label = self.base.get_node_as::<Label>("MessageLabel");
-        message_label.set_text("Dodge the\nCreeps!".into());
+        message_label.set_text("Dodge The Creeps".into());
```

Comment:
Why this change? The line break is for stylistic reasons and exists [in the original](https://github.com/godotengine/godot-demo-projects/blob/master/2d/dodge_the_creeps/HUD.gd#L14), as well.

Reply:
Carryover from my implementation. I set a different screen size and didn't catch a difference. Swapping back and then squashing into a single commit.